### PR TITLE
Bookstore: Redux in React components

### PR DESCRIPTION
**Pull request by keebot-png**
## In this project I implemented the following:
- [x] Use store, actions and reducers in React.
- [x] Add functionality to add button and remove button
- [x] Connect React and Redux.
- [x] Use useSelector to consume the state.
- [x] Used useDispatch to send books and categories the following actions into the redux store
- [x] Fix linter errors

### CODE REVIEW

File: src/components/category/Category.js [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
+
+  return (
+    <div className="status-log">
+      <div>
+        {categories.map((category) => (
+          <p key={category}>
+            {category}
+          </p>
+        ))}
+      </div>
```
Comment (ha-manel) [inline src/components/category/Category.js:19]: - [ ] Great job on making the books section perfectly 👏 for the categories, we're supposed to see a button, when that button is clicked the action creator `underConstruction` is dispatched and we see the state `Under Construction`. However, the app crashes and I get the following error 👇 

image

hint: I think the state slice that you want to get here is `categoriesReducer` and not `reducer` (line 7). 
hint2: The state is a string and not an iterable so you can't use `map` on it 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @keebot-png 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/category/Category.js (hunk 1 above)
Comment (keebot-png) [reply src/components/category/Category.js:19]: @ha-manel Thank you for the timely response and for providing hints. I have corrected the issues and everything seems to be running fine.

Comment (ha-manel) [reply src/components/category/Category.js:19]: awesome! good work 😁

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @keebot-png, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**